src/backend/.config/.env.local

```diff
@@ -14,6 +14,11 @@ POSTGRES_DB = dpia
 POSTGRES_USER = postgres
 POSTGRES_PASSWORD = postgres
 
+################### feature flag setting ###############
```

Comment:
Can you please rename this comment as well

Reply:
done